frontend/src/app/pages/exercises/exercise/shared/editor-panel/vehicle-template-form/vehicle-template-form.component.html

```diff
@@ -110,27 +110,25 @@
             *ngIf="values.materialTypes.length > 0; else noMaterialTemplates"
             class="d-flex flex-wrap"
         >
-            <div
+            <app-map-editor-card
                 *ngFor="
                     let materialTypes of values.materialTypes;
                     let index = index
                 "
-                class="card text-center m-1"
-                style="width: 10rem; text-overflow: ellipsis; height: auto"
+                style="width: 8rem"
             >
-                <div>
-                    <button
-                        (click)="removeMaterial(index)"
-                        class="btn btn-sm btn-outline-danger float-end"
-                    >
-                        <i class="bi bi-trash"></i>
-                    </button>
-                </div>
+                <button
+                    (click)="removeMaterial(index)"
+                    class="btn btn-sm btn-outline-danger position-absolute"
+                    style="top: 0; right: 0"
+                >
+                    <i class="bi bi-trash"></i>
+                </button>
                 <app-material-template-display
                     [materialTemplateType]="materialTypes"
                 >
                 </app-material-template-display>
```

Comment:
Button and material template display have the wrong order on the z-index, the button should overlay the material image, just like the edit button in the editor sidebar

<img width="124" height="107" alt="image" src="https://github.com/user-attachments/assets/0ee2cc41-ccc8-4510-9bb0-23e543c35c58" />

(I haven't checked since the personnel icons are not large enough, but I suppose the personnel cards suffer from the same issue…)